AwardsServer/AwardsServer/ServerUI/UIForm.cs

```diff
@@ -269,13 +269,28 @@ public void UpdateOptions()
             }
         }
 
+        public void SetLocalIP()
+        {
+            try
+            {
+                var path = Options.ServerIP_File_Path;
+                var ip = GetLocalIPAddress();
+                if(!string.IsNullOrWhiteSpace(path) && !string.IsNullOrWhiteSpace(ip))
+                    System.IO.File.WriteAllText(Options.ServerIP_File_Path, GetLocalIPAddress());
```

Comment:
```suggestion
                    System.IO.File.WriteAllText(Options.ServerIP_File_Path, ip);
```

Dont know why im not using the variable i just declared